src/utils/api.js

```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
+
+const newApi = axios.create({
+  baseURL: "https://my-nc-news-pemi.onrender.com/api",
+});
+
+export const getArticles = (articles_types) => {
+  return newApi
+    .get("/articles", {
+      params: {
+        type: articles_types,
```

Comment:
Nice use of Axios, but again, there's no requirement for any params yet at this point.